apps/dg/resources/en/function_strings.json

```diff
@@ -490,34 +490,44 @@
         "description": "Will be evaluated in the context of the next case, return the resulting value."
       },
       {
+        "name": "argument",
+        "type": "filter",
+        "description": "A default parameter if using a filter. If there is no default, the last case evaluates as empty."
```

Comment:
Actually, I believe 'next' and 'prev' have **two** optional arguments: 'default' and 'filter.' The first of these is the value that will be returned if there is no next (or previous) case. The second of these limits the cases that are considered.